LiteLoader/Header/FormUI.h

```diff
@@ -331,6 +346,33 @@ class SimpleForm : public FormImpl {
     LIAPI bool sendTo(Player* player, Callback callback = Callback());
 };
 
+class ModalForm : public FormImpl {
+protected:
+    LIAPI string serialize() override;
+
+public:
+    using Callback = std::function<void(Player*, bool)>;
+    string title, content, button1, button2;
```

Comment:
Change to more specific and canonical naming

Reply:
button1Text and button2Text? Or confirmButton and cancelButton?